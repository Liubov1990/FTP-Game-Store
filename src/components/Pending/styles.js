export const styles = () => ({
  loadingWrapper: {
    display: "flex",
    justifyContent: "center",
    minHeight: "calc(100vh - 76px)"
  },
  root: {
    alignSelf: "center"
  },
  colorPrimary: {
    color: "rgba(0, 0, 0, .5)"
  }
});
