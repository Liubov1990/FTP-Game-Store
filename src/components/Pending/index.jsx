import React from "react";
// material-ui
import { CircularProgress } from "@material-ui/core";
import { makeStyles } from "@material-ui/core/styles";
// styles
import { styles } from "./styles";

const useStyles = makeStyles(styles);

function Pending() {
  const classes = useStyles();

  return (
    <div className={classes.loadingWrapper}>
      <CircularProgress className={`${classes.root} ${classes.colorPrimary}`} />
    </div>
  );
}
export default Pending;
