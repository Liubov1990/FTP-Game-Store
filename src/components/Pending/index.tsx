import React, { ReactElement } from "react";
// material-ui
import { CircularProgress } from "@material-ui/core";
import { makeStyles } from "@material-ui/core/styles";
// styles
import { styles } from "./styles";

const useStyles = makeStyles(styles);

function Pending(): ReactElement {
  const classes = useStyles();

  return (
    <div className={classes.loadingWrapper}>
      <CircularProgress className={`${classes.root} ${classes.colorPrimary}`} />
    </div>
  );
}
export default Pending;
