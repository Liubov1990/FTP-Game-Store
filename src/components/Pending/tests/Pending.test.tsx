import React from "react";
import { render } from "@testing-library/react";
import Pending from "..";

describe("Pending", () => {
  test("should match snapshot", () => {
    const { container } = render(<Pending />);
    expect(container.firstChild).toMatchSnapshot();
  });
});