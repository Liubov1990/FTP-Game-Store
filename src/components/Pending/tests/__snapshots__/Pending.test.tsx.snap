// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pending should match snapshot 1`] = `
<div
  class="makeStyles-loadingWrapper-1"
>
  <div
    class="MuiCircularProgress-root makeStyles-root-2 makeStyles-colorPrimary-3 MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
    role="progressbar"
    style="width: 40px; height: 40px;"
  >
    <svg
      class="MuiCircularProgress-svg"
      viewBox="22 22 44 44"
    >
      <circle
        class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
        cx="44"
        cy="44"
        fill="none"
        r="20.2"
        stroke-width="3.6"
      />
    </svg>
  </div>
</div>
`;
